okta/komand_okta/triggers/users_added_removed_from_group/trigger.py

```diff
@@ -0,0 +1,107 @@
+import komand
+import time
+from .schema import UsersAddedRemovedFromGroupInput, UsersAddedRemovedFromGroupOutput, Input, Output, Component
+# Custom imports below
+import itertools
+from komand.exceptions import PluginException
+
+
+class UsersAddedRemovedFromGroup(komand.Trigger):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='users_added_removed_from_group',
+                description=Component.DESCRIPTION,
+                input=UsersAddedRemovedFromGroupInput(),
+                output=UsersAddedRemovedFromGroupOutput())
+
+    def run(self, params={}):
+        """Run the trigger"""
+        group_list = params.get(Input.GROUP_IDS)
+        okta_url = self.connection.okta_url
+        current_list = list()
+        group_names = list()
+        for group in group_list:
+            api = f"{okta_url}/api/v1/groups/{group}/users"
+            # Build a reference list to check for updates against
+            response = self.connection.session.get(api)
+            if response.status_code in range(400, 499):
+                raise PluginException(cause="Okta returned a 4xx status code",
+                                      assistance="Ensure that the secretKey key is both valid and correct",
+                                      data=response.text)
+            if response.status_code in range(500, 599):
+                raise PluginException(cause="Okta returned a 5xx status code",
+                                      assistance="Ensure that the okta URL key is  correct",
+                                      data=response.text)
+            try:
+                data = response.json()
+                data = komand.helper.clean(data)
+            except ValueError:
+                raise PluginException(cause='Returned data was not in JSON format',
+                                      assistance="Double check that group ID's are all valid",
+                                      data=response.text)
+            current_list.append({group: data})
+
+            # Get group names
+            group_name_api = f"{okta_url}/api/v1/groups/{group}"
+            response = self.connection.session.get(group_name_api)
+            if response.status_code in range(400, 499):
+                raise PluginException(cause="Okta returned a 4xx status code",
```

Comment:
```suggestion
                raise PluginException(cause="Okta returned a 4xx status code.",
```
You'll need a period after all the sentences in a PluginException